Added link to the SciPy 2019 conference talk (#89)
jni/skan: doc/getting_started.rst

But we hope this minimal example will serve for inspiration for your
future analysis of skeleton images.

> If you are interested in how `numba <https://numba.pydata.org/>`_ was used
> to accelerate some parts of Skan, check out my `talk <https://www.youtube.com/watch?v=0pUPNMglnaE>`_
> on the the SciPy 2019 conference.

Reviewer: ```suggestion
If you are interested in how we used `numba <https://numba.pydata.org/>`_
to accelerate some parts of Skan, check out `jni's talk <https://www.youtube.com/watch?v=0pUPNMglnaE>`_
at the the SciPy 2019 conference.
```
